CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Change Log
 
+### 0.17.5 - 2022-02-16
+
+### Fixed
+
+- Fix case where url is fragmented in httptools protocol (#1263) 2/16/22
+- Fix WSGI middleware not to explode quadratically in the case of a larger body (#1329) 2/16/16
```

Comment:
Thanks for merging #1329!
But hey... while I did start working on it a while ago, that wasn't 5 years ago :thinking:

Reply:
/hide
I wasn't born at that time :turtle: !

Reply:
Just since we're here - I don't think we really need dates against individual PRs.
The release date is what matters.

Reply:
agreed!